Cargo.toml

```diff
@@ -8,7 +8,7 @@ authors = ["Tobie Morgan Hitchcock <tobie@surrealdb.com>"]
 
 [features]
 # Public features
-default = ["storage-mem", "storage-rocksdb", "scripting", "http"]
+default = ["storage-mem", "storage-rocksdb", "scripting", "http", "sql2"]
```

Comment:
This isn't necessary and `sql2` is now deprecated in the binary.

Reply:
sorry, this was a mistakenly added to the commit